### DISCUSSION TITLE
[Feature]: Favourites section for assets

### DISCUSSION THREAD

**Discussion by zackpollard** [immich-app/immich#1680] [Feature Request]
### Feature detail

Google photos allows you to favourite images and videos that you want to keep, this is probably most commonly used as a refined gallery as it's very easy to go through and favourite images to reduce the amount of images you have and curate a quality gallery section as you take photos.

### Platform

Web & Mobile

**Comment by bo0tzz** [comment]
Partially implemented in #939, but only on web and still missing a way to view all favourited assets.
